src/QuadForm/Herm/GenusRep.jl

```diff
@@ -393,4 +395,314 @@ function iterated_neighbours(L::HermLat, P; use_auto = false, max = inf,
   return result
 end
 
+@doc Markdown.doc"""
+    neighbours_with_ppower(L::HermLat, P::NfRelOrdIdl, e::Integer, use_auto = true)
+
+Return a representative of an isometry class in the special genus of `L`,
```

Comment:
There is no guarantee that the output lattice lies in the same special genus as `L`.
Rather the way that it is used is exactly to produce a lattice in the genus which is in a different special genus.
That is the purpose of Markus Algorithm 4.7.

It rather outputs a chain of p-neighbors where the next one is different from the previous one.
So that it is indeed a chain and not a loop.

Return a chain of `P` neighbors `of length `e`. 
`L=L_1, L_2 ... , L_e` such that L_{i-1} != L_{i+1} for `i = 2, \dots , e-1`.